pkg/minikube/assets/addons.go

```diff
@@ -440,6 +440,106 @@ var Addons = map[string]*Addon{
 			"0640",
 			false),
 	}, false, "gcp-auth"),
+	"volumesnapshots": NewAddon([]*BinAsset{
+		MustBinAsset(
+			"deploy/addons/volumesnapshots/snapshot.storage.k8s.io_volumesnapshotclasses.yaml",
+			vmpath.GuestAddonsDir,
+			"snapshot.storage.k8s.io_volumesnapshotclasses.yaml",
+			"0640",
+			false),
+		MustBinAsset(
+			"deploy/addons/volumesnapshots/snapshot.storage.k8s.io_volumesnapshotcontents.yaml",
+			vmpath.GuestAddonsDir,
+			"snapshot.storage.k8s.io_volumesnapshotcontents.yaml",
+			"0640",
+			false),
+		MustBinAsset(
+			"deploy/addons/volumesnapshots/snapshot.storage.k8s.io_volumesnapshots.yaml",
+			vmpath.GuestAddonsDir,
+			"snapshot.storage.k8s.io_volumesnapshots.yaml",
+			"0640",
+			false),
+		MustBinAsset(
+			"deploy/addons/volumesnapshots/rbac-volume-snapshot-controller.yaml",
+			vmpath.GuestAddonsDir,
+			"rbac-volume-snapshot-controller.yaml",
+			"0640",
+			false),
+		MustBinAsset(
+			"deploy/addons/volumesnapshots/volume-snapshot-controller-deployment.yaml",
+			vmpath.GuestAddonsDir,
+			"volume-snapshot-controller-deployment.yaml",
+			"0640",
+			false),
+	}, true, "volumesnapshots"),
```

Comment:
Why did you choose to enable this addon by default? Typically, I prefer to keep the number of default addons to a minimum so that a standard cluster doesn't have extra CPU overhead, but if a lot of users will be needing this addon then it could make sense.